airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2218,6 +2218,13 @@ components:
           type: boolean
           default: true
 
+        task_ids:
+          description: A list of task ids to clear.
+          type: array
+          items:
+            type: string
+          minItems: 1
```

Comment:
Does this not imply that task_ids must not be empty?

Reply:
Yes, actually I have two points there:

1) Empty list for `task_ids` can look a little ambiguous. Should it be treated as `null` and no filters should be applied in the case? Either should no tasks be cleared?

2) If no tasks should be cleared in the case of empty list, what is the purpose to call the endpoint?

What do you think?